### PR TITLE
Added E2E test for Image Tensor API

### PR DESCRIPTION
**Pull request by shalvamist**
### Description
Added E2E test - Currently covering -
URL --> Tensor
Tensor --> ImageData
ImageData --> Tensor

Need to add coverage for HTMLImageElement and resize

### CODE REVIEW

File: js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+it('Browser E2E testing - Tensor <--> Image E2E test', async function () {
+  // Image URL to tensor API
+  const inputTensor = await ort.Tensor.fromImage('https://onnxruntime.ai/images/ONNX-Icon.png');
```
Comment (fs-eire) [inline js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js:8]: not sure if this is a good idea - the test is now depending on internet availability.

File: js/web/test/e2e/run.js [hunk 2]
```diff
@@ -109,6 +109,8 @@ async function testAllBrowserCases({ hostInKarma }) {
   await runKarma({ hostInKarma, main: './browser-test-wasm-path-override-filename.js', ortMain: 'ort.wasm.min.js'});
   await runKarma({ hostInKarma, main: './browser-test-wasm-path-override-prefix.js'});
   await runKarma({ hostInKarma, main: './browser-test-wasm-path-override-prefix.js', ortMain: 'ort.wasm.min.js'});
+  await runKarma({ hostInKarma, main: './browser-test-wasm-Image-tensor-Image.js'});
+  await runKarma({ hostInKarma, main: './browser-test-wasm-Image-tensor-Image.js', ortMain: 'ort.wasm.min.js'});
```
Comment (fs-eire) [inline js/web/test/e2e/run.js:113]: as we are not calling into ORT at all, we don't need this one

**Comment by fs-eire** [comment]
please use all lower case for file name

File: js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js (hunk 1 above)
Comment (shalvamist) [reply js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js:8]: Do you suggest we use an asset in the test ? after all it's a URL test API so either we add an asset to the test or we use the internet.

File: js/web/test/e2e/run.js (hunk 2 above)
Comment (shalvamist) [reply js/web/test/e2e/run.js:113]: OK cool I will remove it

File: js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js (hunk 1 above)
Comment (fs-eire) [reply js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js:8]: yes, use an asset and serve it in the E2E is a better idea.

Comment (guschmue) [reply js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js:8]: there are also plenty of png's under onnxruntime/docs

Comment (shalvamist) [reply js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js]: I changed the test to generate white noise image so now we don't rely on internet connection (only in the case of URL -->tensor)

**Review by guschmue — DISMISSED** [review]

**Comment by fs-eire** [comment]
> please use all lower case for file name

the file name still contains upper case

File: js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+function getRndColor() {
+  var r = 255*Math.random()|0,
```
Comment (fs-eire) [inline js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js:7]: never use `var` today. always use `let` or `const`

**Review by fs-eire — APPROVED** [review]

**Pull request merged**